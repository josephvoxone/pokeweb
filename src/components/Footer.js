import React from 'react';
import Logoku from '../assets/joseph.svg';
import styled from '@emotion/styled'

const Footer = () => {

    const FooterComp = styled.div`
        max-width: 1000px;
        align-content: center;
        justify-content: center;
        display: grid;
        margin: 0 auto;
        margin-top: 40px;
        padding: 20px 36px;
        height: 50px;
        background: #2c72b9;
        border-radius: 10px 10px 0 0;
        border-top: 5px solid #ffcb05;

        img{
            max-width: 200px;
            width: 200px;
            margin: 0 auto;
            margin-top: -50px;
        }
    `;

    const Credit = styled.p`
        color: white;
        font-weight: 500;
        font-size: 13px;
    `;

    return (
        <FooterComp>
            <img src={Logoku} width='200' height='79' alt="logoku"></img>
            <Credit>Create with Love and Passion - Project Assignment Tokopedia</Credit>
        </FooterComp>
    )
}

export default Footer;