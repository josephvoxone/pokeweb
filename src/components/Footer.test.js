import React from "react";
import { render, unmountComponentAtNode } from "react-dom";
import { act } from "react-dom/test-utils";
import Footer from './Footer';

let container = null;
beforeEach(() => {
    // setup a DOM element as a render target
    container = document.createElement("div");
    document.body.appendChild(container);
});

afterEach(() => {
    // cleanup on exiting
    unmountComponentAtNode(container);
    container.remove();
    container = null;
});

it("Test : Footer", () => {
    {
        act(() => {
            render(
                <Footer />, container)
        })
        expect(container).toMatchSnapshot()
    }
});
