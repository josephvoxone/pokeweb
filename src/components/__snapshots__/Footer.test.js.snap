// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test : Footer 1`] = `
<div>
  <div
    class="css-8dspe8"
  >
    <img
      alt="logoku"
      height="79"
      src="joseph.svg"
      width="200"
    />
    <p
      class="css-1tlrfr4"
    >
      Create with Love and Passion - Project Assignment Tokopedia
    </p>
  </div>
</div>
`;
